docs/developer/plugin-list.asciidoc

```diff
@@ -513,8 +517,9 @@ As a developer you can reuse and extend built-in alerts and actions UI functiona
 |Registers commercially licensed generic actions like per panel time range and contains some code that supports drilldown work.
 
 
-|{kib-repo}blob/{branch}/x-pack/plugins/upgrade_assistant[upgradeAssistant]
-|WARNING: Missing README.
+|{kib-repo}blob/{branch}/x-pack/plugins/upgrade_assistant/README.md[upgradeAssistant]
+|Upgrade Assistant helps users prepare their Stack for being upgraded to the next major. Its primary
+purposes are to:
```

Comment:
Just FYI... the way this parses the asciidoc files, it only includes the first paragraph it encounters so you can end up with some awkwardness like this.